test/fixtures/linter/autofix/snowflake/003_previously_parse_tree_damaging/after.sql

```diff
@@ -1,2 +1,3 @@
 set cutoff = (select foo
+
```

Comment:
Previously, L036 was deleting this blank line not because we wanted to do this, but simply to avoid a parse error after the fix. Now that `apply_fixes()` is handling this, we can remove the hacky fix from L036, and the newline is automatically preserved, and there's no parse error anymore. 🎉